scss/_icons.scss

```diff
@@ -45,3 +45,19 @@ img.material-icon {
   vertical-align: middle;
   transform: translate(0px, -0.07em);
 }
+
+i.material-svg-icon {
+  -webkit-mask-repeat: no-repeat;
+  background-color: var(--globalText);
+  display: inline-block;
+  height: 1.15em;
+  width: 1.15em;
+  -webkit-mask-size: 1.15em;
+  margin-inline-start: 0.3em;
+  margin-inline-end: 0.3em;
+  vertical-align: middle;
+}
+
+i.material-icon-schedule {
+  -webkit-mask-image: url(../images/icons/schedule.svg);
```

Comment:
```suggestion
  -webkit-mask-image: url(../images/icons/schedule.svg);
  transform: translate(0px, -0.07em);
```

adding this makes it look more centred, I have the values from eh other icons above.

Reply:
![image](https://github.com/deltachat/deltachat-desktop/assets/15038218/2d4fbb37-1a8f-494f-b903-feb3346c0c55)
![image](https://github.com/deltachat/deltachat-desktop/assets/15038218/a64691cf-5538-4d30-bc93-c2f17b7a0215)

It is fine as it is now

The first image is from this PR and the latter from master(with manually whitening the clock icon)

Reply:
I think the offset looks better, I tried it because to me it looked a bit off.